tremor-pipeline/src/op/trickle/select.rs

```diff
@@ -768,7 +823,7 @@ impl Operator for TrickleSelect {
                         aggregates,
                         &group_str,
                         &group_value,
-                    );
+                    )?;
```

Comment:
since this is the hot path, can we use `stry` here or are the error types different?

Reply:
i am gonna stry all the things here!